documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/TrashStorageImpl.java

```diff
@@ -47,25 +48,28 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-import static org.exoplatform.documents.storage.jcr.util.JCRDocumentsUtil.getNodeByIdentifier;
+import static org.exoplatform.documents.storage.jcr.util.NodeTypeConstants.EXO_DATE_MODIFIED;
+import static org.exoplatform.documents.storage.jcr.util.NodeTypeConstants.EXO_TITLE;
 
 public class TrashStorageImpl implements TrashStorage {
 
-  private static final String FILE_EXPLORER_PORTLET = "FileExplorerPortlet";
+  private static final String    FILE_EXPLORER_PORTLET   = "FileExplorerPortlet";
 
-  private static final String TRASH_WORKSPACE = "trashWorkspace";
+  private static final String    TRASH_WORKSPACE         = "trashWorkspace";
 
-  private static final String TRASH_HOME_PATH = "trashHomeNodePath";
+  private static final String    TRASH_HOME_PATH         = "trashHomeNodePath";
 
-  private RepositoryService repositoryService;
+  private static final String    SELECT_FROM_TRASH_QUERY = "SELECT * FROM nt:base WHERE exo:restorePath IS NOT NULL";

```

Comment:
offset and limit must be applied in this query

Reply:
This is just a string used to refactor the code. It is used by multiple methods that do not use the offset and limit. In our case, the offset and limit are applied during the building of the query!

Reply:
:+1: